From e354f8da8109149cb41a2cc98863969dc5e91cb8 Mon Sep 17 00:00:00 2001
From: Garrett Brown <themagnificentmrb@gmail.com>
Date: Thu, 26 Aug 2021 15:52:45 -0700
Subject: [PATCH] Fix unstaged changes breaking build

Error was:

  warning: CRLF will be replaced by LF in src/drivers/jollyjgr.c.
  The file will have its original line endings in your working directory
  fatal: No rebase in progress?
  error: Your local changes to the following files would be overwritten by merge:
          src/drivers/jollyjgr.c
  Please commit your changes or stash them before you merge.
  Aborting
---
 .gitattributes | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/.gitattributes b/.gitattributes
index b354aec..47bda49 100644
--- a/.gitattributes
+++ b/.gitattributes
@@ -1,11 +1,6 @@
 # Set the default behavior, in case people don't have core.autocrlf set.
 * text=auto
 
-# Declare files that will always have LF Unix-style line endings on checkout.
-*.c text eol=lf
-*.h text eol=lf
-*.in text eol=lf
-
 # Denote all files that are truly binary and should not be modified.
 *.png binary
 *.jpg binary
-- 
2.30.2

